SwiftWisdom/Core/DirectoryManager/DirectoryManager.swift

```diff
@@ -11,7 +11,10 @@ import Foundation
 public final class DirectoryManager {
     
     enum Error : ErrorType {
-        case UnableToCreatePath(url: NSURL?) // TODO: When upgrading to 2.3 Swift, this type changed from NSURL -> NSURL?. Review to see if this is an appropriate change.
+        // TODO: When upgrading to 2.3 Swift, this type changed from NSURL -> NSURL?. 
```

Comment:
I don't have the bandwidth to make to do a full review on this...unless colden wants to pick this up. Are you saying that it is acceptable to make the change to NSURL?